### PR TITLE
user experience classic ready

### PR DESCRIPTION
**Pull request by Amet015**

### CODE REVIEW

File: src/test/resources/features/Opportunity.feature [hunk 1]
```diff
@@ -63,4 +63,3 @@ Feature: Opportunity
   Scenario: Create a new opportunity
     Given I create 2 "Basic" opportunities
     When I delete created opportunities
-
```
Comment (JuanCarlosMartinez4) [inline src/test/resources/features/Opportunity.feature:66]: add  a new line please

File: src/main/java/salesforce/ui/pages/opportunity/OpportunityClassicPage.java [hunk 2]
```diff
@@ -154,6 +161,14 @@ public void clickOnNotesAndAttachmentsButton() {
...
     public AbstractTaskOpportunity clickAddTask() {
-        return null;
+        webDriverWait.until(ExpectedConditions.elementToBeClickable(addTaskButton));
+        webDriverWait.until(ExpectedConditions.visibilityOf(addTaskButton));
+        webDriverWait.until(ExpectedConditions.elementToBeClickable(editButton));
+        webDriverWait.until(ExpectedConditions.elementToBeClickable(addFile));
+        webDriverWait.until(ExpectedConditions.visibilityOf(addFile));
+        addTaskButton.click();
+        addTaskButton.click();
+        addTaskButton.click();
```
Comment (JuanCarlosMartinez4) [inline src/main/java/salesforce/ui/pages/opportunity/OpportunityClassicPage.java]: is it the correct? and could be better if you have a method to do this action.
For example:
private void clickOnAddTaskButton() {
      webDriverWait.until(ExpectedConditions.visibilityOf(addTaskButton));
        addTaskButton.click();
}
**In that case you are using Single responsability**

File: src/main/java/salesforce/ui/pages/task/TaskClassicPage.java [hunk 3]
```diff
@@ -9,55 +9,93 @@
...
 package salesforce.ui.pages.task;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+
 /**
  * Defines an TaskClassicPage.
```
Comment (JuanCarlosMartinez4) [inline src/main/java/salesforce/ui/pages/task/TaskClassicPage.java]: Defines **a** TaskClassicPage

File: src/main/java/salesforce/ui/pages/opportunity/OpportunityClassicPage.java (hunk 2 above)
Comment (Amet015) [reply src/main/java/salesforce/ui/pages/opportunity/OpportunityClassicPage.java]: yes but that button is unavialable. That's why I am using it.

Comment (JuanCarlosMartinez4) [reply src/main/java/salesforce/ui/pages/opportunity/OpportunityClassicPage.java]: ok, but implement this please
private void clickOnAddTaskButton() {
webDriverWait.until(ExpectedConditions.visibilityOf(addTaskButton));
addTaskButton.click();
}

Comment (Amet015) [reply src/main/java/salesforce/ui/pages/opportunity/OpportunityClassicPage.java]: that doesn't wok, because it does't find the element

Comment (Amet015) [reply src/main/java/salesforce/ui/pages/opportunity/OpportunityClassicPage.java]: I separate it.

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Pull request merged**